specs/Checkbox.md

```diff
@@ -0,0 +1,388 @@
+| as                   | React.ElementType                                           | default type is "div"                                                          |
+| className            | string                                                      |                                                                                |
+| content              | ReactNode                                                   |                                                                                |
+| design               | ComponentDesign                                             |                                                                                |
+| disableAnimations    | boolean                                                     | default false                                                                  |
+| overwrite            | boolean                                                     | default false                                                                  |
+| renderer             | Renderer                                                    |                                                                                |
+| rtl                  | boolean                                                     | default false                                                                  |
+| styles               | ComponentSlotStyle                                          |                                                                                |
+| target               | Document                                                    |                                                                                |
+| theme                | ThemeInput                                                  |                                                                                |
+| variables            | any                                                         |                                                                                |
+
+### Differences of Fabric/Stardust to resolve
+
+| Name                 | Type                                                        | Notes                                                                          |
+| -------------------- | --------------------------------------------------------    | -------------------------------------------------------------------------------|
+| animation            | AnimationProp                                               |                                                                                |
+| disableAnimations    | boolean                                                     | default false                                                                  |
+| overwrite            | boolean                                                     | default false                                                                  |
```

Comment:
overwrite, renderer, target, content disableAnimations aren't in stardust or fabric.... can be removed